src/styles/blocks/header.scss

```diff
@@ -0,0 +1,66 @@
+.header {
+  background-color: $color-white;
+  padding: 0 50px;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+  min-width: 900px;
+
+  &__inner {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+  }
+
+  &__menu {
+    text-transform: uppercase;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 14px;
+  }
+  
+  &__li {

```

Comment:
Don't use the naming which duplicates tag names.
It's better to name elements with whole words: `__list`, `__link`, etc.